openmetadata-ui/src/main/resources/ui/src/pages/MyDataPage/MyDataPage.component.tsx

```diff
@@ -40,13 +41,17 @@ const MyDataPage = () => {
   const [searchResult, setSearchResult] = useState<SearchResponse>();
   const [entityCounts, setEntityCounts] = useState<EntityCounts>();
 
-  const fetchData = (value: SearchDataFunctionType, fetchService = false) => {
+  const fetchData = (
+    value: SearchDataFunctionType,
```

Comment:
@darth-coder00 `SearchDataFunctionType` has `size` as an optional field so we can use that instead of passing `pageSize` directly.